app/models/node.rb

```diff
@@ -94,6 +94,11 @@ def updated_month
   validates :title, presence: :true
   validates_with UniqueUrlValidator, on: :create
 
+  scope :status_one, -> { where(status: 1) }
```

Comment:
what if we renamed this like "approved" or "published" since that's what it means?